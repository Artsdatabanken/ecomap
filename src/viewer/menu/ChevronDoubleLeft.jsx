import React from 'react'
import SvgIcon from 'material-ui/SvgIcon'

function ChevronDoubleLeft () {
  return (
    <SvgIcon viewBox='00 00 2000 1000'>
      <path d='M578.6,500l386-385.9c6.5-6.5,9.8-14.1,9.8-22.6c0-8.5-3.3-16-9.8-22.6l-49.1-49.1c-6.5-6.5-14.1-9.8-22.6-9.8c-8.5,0-16,3.3-22.6,9.8L412.6,477.4c-6.5,6.5-9.8,14.1-9.8,22.6c0,8.5,3.3,16,9.8,22.6l457.6,457.6c6.5,6.6,14.1,9.8,22.6,9.8c8.5,0,16-3.3,22.6-9.8l49.1-49.1c6.5-6.5,9.8-14.1,9.8-22.6c0-8.5-3.3-16-9.8-22.6L578.6,500z' />
      <path d='M201.5,500l385.9-385.9c6.5-6.5,9.8-14.1,9.8-22.6c0-8.5-3.3-16-9.8-22.6l-49.1-49.1c-6.5-6.5-14.1-9.8-22.6-9.8c-8.5,0-16.1,3.3-22.6,9.8L35.5,477.4c-6.5,6.5-9.8,14.1-9.8,22.6c0,8.5,3.3,16,9.8,22.6l457.6,457.6c6.5,6.6,14.1,9.8,22.6,9.8c8.5,0,16-3.3,22.6-9.8l49.1-49.1c6.5-6.5,9.8-14.1,9.8-22.6c0-8.5-3.3-16-9.8-22.6L201.5,500z' />
    </SvgIcon>
  )
}

export default ChevronDoubleLeft
