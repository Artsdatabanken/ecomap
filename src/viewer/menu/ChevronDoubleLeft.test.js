import React from 'react'
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
import renderer from 'react-test-renderer'
import muiTheme from '../../theme_jest'

import ChevronDoubleLeft from './ChevronDoubleLeft'

it('renders ok', () => {
  const tree = renderer.create(
    <MuiThemeProvider muiTheme={muiTheme}>
      <ChevronDoubleLeft />
    </MuiThemeProvider>)
  expect(tree).toMatchSnapshot()
})
