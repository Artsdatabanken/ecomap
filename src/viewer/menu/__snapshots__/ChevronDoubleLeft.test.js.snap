// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ok 1`] = `
<svg
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  style={
    Object {
      "color": "rgba(0, 0, 0, 0.87)",
      "display": "inline-block",
      "fill": "currentColor",
      "height": 24,
      "muiPrepared": true,
      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
      "userSelect": "none",
      "width": 24,
    }
  }
  viewBox="00 00 2000 1000"
>
  <path
    d="M578.6,500l386-385.9c6.5-6.5,9.8-14.1,9.8-22.6c0-8.5-3.3-16-9.8-22.6l-49.1-49.1c-6.5-6.5-14.1-9.8-22.6-9.8c-8.5,0-16,3.3-22.6,9.8L412.6,477.4c-6.5,6.5-9.8,14.1-9.8,22.6c0,8.5,3.3,16,9.8,22.6l457.6,457.6c6.5,6.6,14.1,9.8,22.6,9.8c8.5,0,16-3.3,22.6-9.8l49.1-49.1c6.5-6.5,9.8-14.1,9.8-22.6c0-8.5-3.3-16-9.8-22.6L578.6,500z"
  />
  <path
    d="M201.5,500l385.9-385.9c6.5-6.5,9.8-14.1,9.8-22.6c0-8.5-3.3-16-9.8-22.6l-49.1-49.1c-6.5-6.5-14.1-9.8-22.6-9.8c-8.5,0-16.1,3.3-22.6,9.8L35.5,477.4c-6.5,6.5-9.8,14.1-9.8,22.6c0,8.5,3.3,16,9.8,22.6l457.6,457.6c6.5,6.6,14.1,9.8,22.6,9.8c8.5,0,16-3.3,22.6-9.8l49.1-49.1c6.5-6.5,9.8-14.1,9.8-22.6c0-8.5-3.3-16-9.8-22.6L201.5,500z"
  />
</svg>
`;
